### ISSUE TITLE
Suggestion: add the first four bytes from the stack top to the trace

### ISSUE THREAD

**Issue by Ptival** [robert/gameboy-doctor#26]
I'm currently debugging a problem with my emulator where it appears we don't pop the same value from the stack.

This is in general not super practical to debug as I don't know how far before we started disagreeing on the stack state. I think it'd be nice to have a small fragment of the stack on the trace.

**Comment by robert** [comment]
Sounds reasonable, want to try a PR? Maybe with a `--with_stack` argument so people can choose not to use this.
